client/post-editor/drafts-button/index.jsx

```diff
@@ -10,12 +10,12 @@ import { translate } from 'i18n-calypso';
  */
 import Button from 'components/button';
 import Count from 'components/count';
-import QueryPostCounts from 'components/data/query-post-counts';
+import QueryPosts from 'components/data/query-posts';
 import { getSelectedSiteId } from 'state/ui/selectors';
 import { isJetpackSite } from 'state/sites/selectors';
-import { getAllPostCount } from 'state/posts/counts/selectors';
```

Comment:
> I'm not sure I understand why the selector's count can not represent the right value since we use this same query to fetch the drafts to show.

Because I could have more drafts than what is shown, since posts are retrieved in batches.

> the drafts shown just under the button can contain more than "my drafts" only

What I mean is we could switch to the same system `/posts` have, with a toggle to see your drafts and everyone's drafts.

The Editor drafts list is mostly an experiment for quick access to the things you are working on, but it's lagging behind "/posts" now.